moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js

```diff
@@ -75,46 +76,45 @@ const ProjectSummaryProjectDescription = ({
    * Updates the description state
    * @param {Object} e - Event object
    */
-  const handleProjectDescriptionChange = e => {
+  const handleProjectDescriptionChange = (e) => {
     setDescription(e.target.value);
   };
 
   return (
     <Grid item xs={12} className={classes.fieldGridItem}>
-      <Typography className={classes.fieldLabel}>Project description</Typography>
+      <Typography className={classes.fieldLabel}>Description</Typography>
       <Box
         display="flex"
         justifyContent="flex-start"
         className={classes.fieldBox}
+        flexWrap="nowrap"
+        alignItems="center"
       >
         {editMode && (
           <>
             <TextField
+              autoFocus
               fullWidth
               multiline={true}
-              rows={4}
+              minRows={4}
               id="moped-project-description"
               label={null}
               onChange={handleProjectDescriptionChange}
               value={description}
             />
-            <Icon
-              className={classes.editIconConfirm}
-              onClick={handleProjectDescriptionSave}
-            >
-              check
-            </Icon>
-            <Icon
-              className={classes.editIconConfirm}
-              onClick={handleProjectDescriptionClose}
-            >
-              close
-            </Icon>
+
+            <IconButton onClick={handleProjectDescriptionSave}>
+              <Icon>check</Icon>
+            </IconButton>
+
+            <IconButton onClick={handleProjectDescriptionClose}>
+              <Icon>close</Icon>
+            </IconButton>
           </>
         )}
         {!editMode && (
           <ProjectSummaryLabel
-            text={description || "None"}
+            text={description || " - "}
```

Comment:
Please ignore me if I'm an outlier with this opinion, but I feel like "None" is more informative than `-` and is clearer to the user.

Reply:
personally i like " - ", it feels a lot cleaner to me from a design perspective and indicates that the field is blank/hasn't been filled in yet.